src/router/index.ts

```diff
@@ -21,6 +21,15 @@ const routes = [
       requiresAuth: false,
     },
   },
+  {
+    path: "/InstructionPage",
```

Comment:
no -- we don't want to provide this endpoint. i dont think this is required in your code, too? correct?